### PR TITLE
Ubuntu22

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

File: templates/ubuntu22/common/base.dockerfile.j2 [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    rm -Rf install/bin install/etc/samples
+
+WORKDIR /opt/repo/opencv/build/install
+CMD ["/bin/bash"]
+# -------------------------------------------------------------------------------------------------
+{% endif %}
+
+FROM ubuntu:22.04 AS ov_base
+
+LABEL description="This is the {{ distribution }} image for {{ product_name }} on Ubuntu 20.04 LTS"
```
Comment (dkalinowski) [inline templates/ubuntu22/common/base.dockerfile.j2:159]:
```suggestion
LABEL description="This is the {{ distribution }} image for {{ product_name }} on Ubuntu 22.04 LTS"
```

File: templates/ubuntu22/common/base.dockerfile.j2 [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+{% elif 'dev' == distribution or 'dev_no_samples' == distribution %}
+ARG LGPL_DEPS="g++ \
+               gcc \
+               libc6-dev"
+ARG INSTALL_PACKAGES="-c=opencv_req -c=python -c=cl_compiler -c=core -c=dev"
+{% else %}
+ARG LGPL_DEPS="g++ \
+               gcc \
+               libc6-dev"
+ARG INSTALL_PACKAGES="-c=opencv_req -c=python -c=cl_compiler -c=opencv_opt -c=dlstreamer"
```
Comment (dkalinowski) [inline templates/ubuntu22/common/base.dockerfile.j2:199]: dlstreamer?

File: templates/ubuntu22/common/base.dockerfile.j2 [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+	  if [ $exit_status -eq 0 ]; then \
+	    apt-get source -q --download-only $package;  \
+	  fi \
+      done && \
+      echo "Download source for $(ls | wc -l) third-party packages: $(du -sh)"; fi && \
+    rm /usr/lib/python3.*/lib-dynload/readline.cpython-3*-gnu.so && rm -rf /var/lib/apt/lists/*
+
+RUN curl -L -O  https://github.com/oneapi-src/oneTBB/releases/download/v2021.9.0/oneapi-tbb-2021.9.0-lin.tgz && \
+    tar -xzf  oneapi-tbb-2021.9.0-lin.tgz&& \
+    cp oneapi-tbb-2021.9.0/lib/intel64/gcc4.8/libtbb.so* /opt/intel/openvino/runtime/lib/intel64/ && \
```
Comment (dkalinowski) [inline templates/ubuntu22/common/base.dockerfile.j2:232]: is it the same tbb that is enforced to use by OV?

File: dockerfiles/ubuntu22/third-party-programs-docker-dev.txt [hunk 4]
```diff
@@ -0,0 +1,3248 @@
...
+17.libtinfo5
+
+This is the Debian prepackaged version of the ncurses
+library and terminfo utilities. ncurses/terminfo was originally written
+by Pavel Curtis and Zeyd M. Ben-Halim <zmbenhal@netcom.com>, and is
+currently held by the Free Software Foundation.
+
+This package was put together by Vaidhyanathan G Mayilrangam
+<vaidhy@debian.org> and Joel Klecker <espy@debian.org>, using sources
+obtained from ftp://ftp.gnu.org/gnu/ncurses/ncurses-5.0.tar.gz.
```
Comment (dkalinowski) [inline dockerfiles/ubuntu22/third-party-programs-docker-dev.txt:2645]: is this list of licenses still actual for ubuntu22 in 2023.0?

Comment (dtrawins) [reply dockerfiles/ubuntu22/third-party-programs-docker-dev.txt:2645]: it should be the same like in ubuntu20, we might rescan the licenses as a separate change

**Review by dkalinowski — APPROVED** [review]

**Pull request merged**